ttnn/cpp/ttnn/operations/eltwise/binary_backward/device/binary_backward_op.hpp

```diff
@@ -166,26 +189,31 @@ struct OpHandler_binary_bw_float<BinaryBackwardOpType::LERP_BW> {
 // Template functions to get the function pointers
 template <BinaryBackwardOpType OpType>
 auto get_function_binary_bw() {
-    return &OpHandler_binary_bw<OpType>::handle;
+    return &OpHandler<OpType>::handle;
 }
 
 template <BinaryBackwardOpType OpType>
 auto get_function_binary_bw_opt_float_default() {
-    return &OpHandler_binary_bw_opt_float_default<OpType>::handle;
+    return &OpHandler<OpType>::handle;
 }
 
 template <BinaryBackwardOpType OpType>
 auto get_function_binary_bw_float_default() {
-    return &OpHandler_binary_bw_float_default<OpType>::handle;
+    return &OpHandler<OpType>::handle;
+}
+
+template <BinaryBackwardOpType OpType>
+auto get_function_binary_bw_string_default() {
+    return &OpHandler<OpType>::handle;
 }
 
 template <BinaryBackwardOpType OpType>
 auto get_function_binary_bw_int_default() {
-    return &OpHandler_binary_bw_int_default<OpType>::handle;
+    return &OpHandler<OpType>::handle;
 }
 
 template <BinaryBackwardOpType OpType>
 auto get_function_binary_bw_float() {
```

Comment:
This function is unnecessary.
You can just use OpHandler<OpType>::handle directly in the op.